pkg/issuer/venafi/client/venaficlient.go

```diff
@@ -352,12 +372,25 @@ func (v *Venafi) VerifyCredentials() error {
 			}
 
 			return nil
-		}
-
-		if v.config.Credentials.User != "" && v.config.Credentials.Password != "" {
-			err := v.tppClient.Authenticate(&endpoint.Authentication{
+		} else if v.config.Credentials.User != "" && v.config.Credentials.Password != "" {
```

Comment:
I would leave this as a simple `if` block, rather than chaining conditionals with an `else if`.

The only reason to have an `else if` here would be if we could escape the previous conditional block if the condition was true. Here, if we have an `AccessToken`, it always returns - we're never getting to the User/Password check no matter what.

Reply:
Thanks for looking. I did this thinking it fixed something.... but probably did not, let me revert and test.

Reply:
That's fair enough, I have reverted back to the two `if` setups in this instance, like it was before.
Something else was not working at the time and I thought this resolved it, but after more testing I don't think it makes any impact on function.